### PR TITLE
Add documentation of how stored incidents are used

### PR DESCRIPTION
**Pull request by fabianvf**

### CODE REVIEW

File: docs/solvedIncidentStore.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Approach to Stored Analyses and Solved Incidents
+
+## Objective
+
+To ingest and permanently store all reports from Kai, using the data to determine and store solutions for fixed incidents, and to support future data extraction and fine-tuning.
```
Comment (jwmatthews) [inline docs/solvedIncidentStore.md:5]: Should this be "store all reports from Konveyor"?

File: docs/solvedIncidentStore.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+2. **Incident Fix Detection**
+3. **Diff Generation and Storage**
+4. **Solution Validation**
+5. **Long-term Data Storage**
+
+## Component Descriptions
+
+### Ingestion of Analysis Reports
+
+All analysis reports generated by Kai are ingested and stored permanently. This ensures that all data is retained for future reference and analysis.
```
Comment (jwmatthews) [inline docs/solvedIncidentStore.md:21]: Similar to prior question, should this be Konveyor?

File: docs/solvedIncidentStore.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+### Incident Fix Detection
+
+The recorded incidents in each analysis report, along with commit information, are used to determine whether an incident was fixed between two reports. This comparison helps identify changes and resolved incidents.
+
```
Comment (jwmatthews) [inline docs/solvedIncidentStore.md:26]: Is it worth calling out how we determine an incident has been fixed and didn't just relocate to a different line number? ... even if we are not yet doing this, just calling out we know this kind of problem exists and is an area to improve could help.

i.e., I assume for an analysis it's likely that there is an unrelated code change so the lineNumber changes, yet its in fact the same incident, just relocated.

File: docs/solvedIncidentStore.md (hunk 1 above)
Comment (fabianvf) [reply docs/solvedIncidentStore.md:5]: whoops yes

**Review by jwmatthews — APPROVED** [review]

**Pull request merged**